aptos-move/framework/aptos-stdlib/sources/simple_map.spec.move

```diff
@@ -0,0 +1,63 @@
+/// Specifications of the `simple_map` module.
+spec aptos_std::simple_map {
+
+    // Make most of the public API intrinsic. Those functions have custom specifications in the prover.
+
+    spec SimpleMap {
+        pragma intrinsic = map,
+            map_new = create,
+            map_len = length,
+            map_destroy_empty = destroy_empty,
+            map_has_key = contains_key,
+            map_add_no_override = add,
+            map_del_return_key = remove,
+            map_borrow = borrow,
+            map_borrow_mut = borrow_mut,
+            map_spec_get = spec_get,
+            map_spec_set = spec_add,
+            map_spec_del = spec_remove,
+            map_spec_has_key = spec_contains_key;
+    }
+
+    spec length {
+        pragma intrinsic;
```

Comment:
Do we still need these `pragma intrinsic` here given the `length` function is already declared intrinsic above?

Reply:
Hmm... without it, Prover outputs the following error: 
```
Error: [internal] boogie exited with compilation errors:
output.bpl(15178,24): error: ";" expected
1 parse errors detected in output.bpl
```
It's because it tries to verify the `length` function although it doesn't have any spec.

Reply:
Maybe there is a bug in the prover?

Reply:
I will open a github issue about this once this PR gets merged.